docs/rules/indent.md

```diff
@@ -72,6 +72,7 @@ This rule has an object option:
 
 * `"SwitchCase"` (default: 0) enforces indentation level for `case` clauses in `switch` statements
 * `"VariableDeclarator"` (default: 1) enforces indentation level for `var` declarators; can also take an object to define separate rules for `var`, `let` and `const` declarations.
+* `"IndentOuterIIFE"` (default: true) when false enforces no indentation level increase for file-level IIFEs.
```

Comment:
We probably need something different here. The other options are all node types, and this is not. Also, we are really talking about indenting the IIFE body, not the IIFE itself, so we need to work on this naming a bit.